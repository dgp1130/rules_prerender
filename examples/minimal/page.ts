import { bar, foo } from './dep';

/** Renders the page. */
export default function(): string {
    return `
<!DOCTYPE html>
<html>
    <head>
        <title>Minimal</title>
        <!-- For live-reload script from \`ts_devserver()\`. -->
        <script src="/_/ts_scripts.js" async defer></script>
    </head>
    <body>
        <h2 id="hello">Hello, World!</h2>
        <span id="foo">${foo}</span>
        <span id="bar">${bar}</span>
    </body>
</html>
    `.trim() + '\n' /* trailing newline */;
}
