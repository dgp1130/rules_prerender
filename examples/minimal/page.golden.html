<!DOCTYPE html>
<html>
    <head>
        <title>Minimal</title>
        <!-- For live-reload script from `ts_devserver()`. -->
        <script src="/_/ts_scripts.js" async defer></script>
    </head>
    <body>
        <h2>Hello, World!</h2>
        <span>foo</span>
        <span>bar</span>
    </body>
</html>
